imports/api/profiles.js

```diff
@@ -1,23 +1,115 @@
-import { Mongo } from 'meteor/mongo';
+import {Mongo} from 'meteor/mongo';
 import SimpleSchema from 'simpl-schema';
 
-export const Profiles = new Mongo.Collection('profiles');
-
-ProfileSchema = new SimpleSchema({
-  _id: Number,
-  firstName: String,
-  lastName: String,
-  username: String,
-  password: String,
-  gender: String,
-  address: String,
-  city: String ,
-  state: String,
-  postalCode: Number,
-  country: String,
-  phone: String,
-  email: String,
-  status: String,
-});
-
-// Profiles.attachSchema(ProfileSchema);
\ No newline at end of file
+Profiles = new Mongo.Collection('profiles');
+
+if (Meteor.isServer) {
+    Profiles.allow({
+        insert: function (userId, doc) {
+            return false;
+        },
+
+        update: function (userId, doc, fieldNames, modifier) {
+            return false;
+        },
+
+        remove: function (userId, doc) {
+            return false;
+        }
+    });
+
```

Comment:
Why are we allowing and denying the same functions? (lines 7-19 vs. lines 21-33)